bin/morgue.js

```diff
@@ -7437,6 +7438,136 @@ async function metricsImporterCmd(argv, config) {
   await cli.routeMethod(argv);
 }
 
+function stabilityCreateMetric(coroner, argv, config) {
+  let universe = argv.universe;
+  if (!universe) {
+    universe = Object.keys(config.config.universes)[0];
+  }
+
+  let project = argv.project;
+
+  if (!universe) {
+    errx("--universe is required");
+  }
+
+  if (!project) {
```

Comment:
nit
does it make sense to assign argv.project to project and validate project name after universe name validation? It's not that important however it will be better if we will 
1. validate universe 
```
if (!universe) {
    errx("--universe is required");
  }
```
2. assign project
3. validate project

Reply:
Moved this. Note that when we do alerts I intend to factor this out to a common function, though I don't know if I'll backport that to this code.